.github/workflows/deploy-vercel-staging.yml

```diff
@@ -435,34 +394,26 @@ jobs:
           name: videos
           path: cypress/videos/
 
-      # We need to find the PR id. Will be used later to comment on that PR.
-      - name: Finding Pull Request ID
-        uses: jwalton/gh-find-current-pr@v1 # See https://github.com/jwalton/gh-find-current-pr
-        id: pr_id_finder
-        if: always() # Forces the job to be always executed, even if a previous job fail.
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-
       # On E2E failure, add a comment to the PR with additional information, if there is an open PR for the current branch
       - name: Comment PR (E2E failure)
         uses: peter-evans/create-or-update-comment@v1 # See https://github.com/peter-evans/create-or-update-comment
-        if: steps.pr_id_finder.outputs.number && failure()
+        if: fromJson(steps.restore-variable.outputs.variables).GITHUB_PULL_REQUEST_ID && success()
```

Comment:
@Demmonius I've removed the step `Finding Pull Request ID`, but I'm not sure whether I should use `fromJson(steps.restore-variable.outputs.variables).GITHUB_PULL_REQUEST_ID && success()` or `GITHUB_PULL_REQUEST_ID && success()` here 🤔 

Can you review?

Reply:
It looks like to be right. What are you thinking about? `GITHUB_PULL_REQUEST_ID` isn't declare right?